src/__test__/Home.test.js

```diff
@@ -0,0 +1,20 @@
+// import { render, screen } from "@testing-library/react"
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.